styles.scss

```diff
@@ -1,7 +1,98 @@
+// FONT
+@import url("https://fonts.googleapis.com/css2?family=Fredoka+One&display=swap");
+
+//VARIABLES
+@import "./variables/mixins";
+@import "./variables/colors";
+
 html,
 body {
   margin: 0;
   padding: 0;
   font-size: 10px;
-  font-family: "Courier New", Courier, monospace;
+  font-family: "Fredoka One", cursive;
+  overflow: hidden;
+}
+
+.main {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+
+  width: 100vw;
+  height: 100vh;
+
+  .play {
+    @include flexCenter;
+
+    width: 100vw;
+    height: 100vh;
+
+    font-size: 8rem;
+
+    display: none;
+  }
+  .top-sector {
+    @include flexCenter;
+
+    width: 100vw;
+    height: 10rem;
+    .app-title {
+      font-size: 4rem;
+      color: $black;
+    }
+  }
+  .middle-sector {
+    @include flexCenter;
+
+    width: 100vw;
+    height: 50rem;
+    display: flex;
+    justify-content: space-around;
+    img {
+      @include imageSpecs;
+    }
+    img:hover {
+      scale: 190%;
+    }
+
+    .rock:hover {
+      animation: shake 0.7s infinite;
+    }
+    .paper:hover {
+      animation: shake 0.7s infinite;
+    }
+    .scissor:hover {
+      animation: shake 0.7s infinite;
+    }
+    .gnarly {
+      display: none;
+    }
+    @keyframes shake {
+      0% {
+        transform: translate(0px, 0px) rotate(0deg);
+      }
+      25% {
+        transform: translate(1px, 1px) rotate(1deg);
+      }
+      50% {
+        transform: translate(-1px, -1px) rotate(-1deg);
+      }
+      75% {
+        transform: translate(2px, -1px) rotate(2deg);
+      }
+      100% {
+        transform: translate(-1px, 1px) rotate(-1deg);
+      }
+    }

```

Comment:
Keyframes can be extrapolated to another file

Reply:
Just like the mixins, por ejemplo?

Reply:
Exactly, maybe into an `animations.scss` file and then import it here